packages/components/src/Actions/ActionButton/ActionButton.component.js

```diff
@@ -89,23 +91,25 @@ function ActionButton(props) {
 
 	const buttonProps = getPropsFrom(Button, rest);
 	const style = link ? 'link' : bsStyle;
-	const rClick =
-		onClick &&
-		(event =>
+	let rClick = null;
+	let rMouseDown = null;
+
+	if (!overlayComponent) {
+		rClick = event =>
```

Comment:
onClick may not be defined in the old version

Reply:
rclick and mousedown are removed just for the overlayComponent. For the rest, we don't change the behavior

Reply:
but here the behavior has changed !

Reply:
overlayComponent is a new feature. It doesn't exist for the old version. So, rclick and mousedown are identic to  the old behavior